ansible/site.yml

```diff
@@ -0,0 +1,45 @@
+- hosts: all
+  sudo: true
+  environment: proxy_env
+  tasks:
+    - hostname: "name={{ hostname }}"
+    - template: src=templates/envvar.sh.j2 dest=/etc/profile.d/envvar.sh owner=root group=root mode=0755
+    - apt: "update_cache=yes name={{ item }}"
+      with_items:
+        - unzip
+        - vim-nox
+        - curl
+        - python-software-properties
+        - git 
+    - shell: creates=/usr/local/go/bin/go curl -L https://storage.googleapis.com/golang/go1.4.2.linux-amd64.tar.gz | tar -xvz -C /usr/local
```

Comment:
ah, sounds good!

I wasn't sure as I saw the diffs that replace the vagrant box image with `ubuntu/vivid64` above. Is that a pre-configured box or a standard box? If not, we need to replace it with our pre-configured box, right?